services/src/unit-test/java/uk/gov/moj/sdt/services/messaging/MessageWriterTest.java

```diff
@@ -45,13 +39,16 @@ class MessageWriterTest extends AbstractSdtUnitTestBase {
 
     private static final String NOT_EXPECTED_TO_FAIL = "Not Expected to fail";
 
+    @Mock
+    JmsTemplate mockJmsTemplate;
+
     /**
      * Set up the variables.
      */
     @BeforeEach
     @Override
     public void setUp() {
-        JmsTemplate jmsTemplate = mock(JmsTemplate.class);
+        JmsTemplate jmsTemplate = mockJmsTemplate;
```

Comment:
This variable is redundant can be removed.  Just use the mockJmsTemplate field directly on line 55.